<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusVirtualPlatfromDialog</class>
 <widget class="QDialog" name="MotusVirtualPlatfromDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>400</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>400</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>400</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>400</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>400</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot6">
   <property name="geometry">
    <rect>
     <x>904</x>
     <y>330</y>
     <width>400</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>650</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>626</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下位机IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="controlLineEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>622</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>192.168.0.125</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="platfromlineEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>700</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>192.168.0.131</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>705</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上位机IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>625</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>704</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="controlPortlineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>621</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="platfromPortlineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>700</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>626</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>接收个数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>706</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送个数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recvCountlineEdit">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>620</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendCountlineEdit">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>700</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>620</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平台状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>620</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sonlineEdit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>700</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mainlineEdit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>626</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>主指令</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>706</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>子指令</string>
   </property>
  </widget>
  <widget class="QComboBox" name="statusBox">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>620</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
